### DISCUSSION TITLE
Topics

### DISCUSSION THREAD

**Discussion by selcuktatar** [findmentor-network/find-mentor#18] [Ideas]
Yazilim Dilleri, Frontend, Backend, Devops, Security gibi topicler yapabiliriz veya arama yaparken sadece isim yerine interest tarafnida arayabilir.

**Comment by cagataycali** [comment]
Aramayi tasarlarken nuxt content'in sagladigi full text search'i kullanabiliriz: https://content.nuxtjs.org/snippets#search

**Comment by cagataycali** [comment]
Bu su an yayinda var!
